theano/tensor/tests/test_basic.py

```diff
@@ -1930,6 +1930,46 @@ def _numpy_true_div(x, y):
                         bad_build=dict(),
                         bad_runtime=dict(bad1=(rand(5, 7), rand(5, 7)),
                                          bad2=(rand(5, 7), rand(8, 3))))
+BatchedDotTester = makeTester(
```

Comment:
This part I will have to look in more details because I am not familiar with makeTester()